src/MgvOfferMaking.sol

```diff
@@ -27,42 +27,32 @@ contract MgvOfferMaking is MgvHasOffers {
     Offer oldOffer;
   }
 
-  /* The function `newOffer` is for market makers only; no match with the existing book is done. A maker specifies how much `inbound_tkn` it `wants` and how much `outbound_tkn` it `gives`.
+  /* The function `newOffer` is for market makers only; no match with the existing offer list is done. The maker specifies how much `olKey.outbound` token it `gives`, and at which `tick` (which is a power of 1.0001 and induces a price). Note that if `tickSpacing > 1`, the actual tick of the offer may differ from the `tick` argument by up to `tickSpacing-1`.
 
      It also specify with `gasreq` how much gas should be given when executing their offer.
 
-     `gasprice` indicates an upper bound on the gasprice at which the maker is ready to be penalised if their offer fails. Any value below Mangrove's internal `gasprice` configuration value will be ignored.
+     `gasprice` indicates an upper bound on the gasprice (in mwei) at which the maker is ready to be penalised if their offer fails. Any value below Mangrove's internal `gasprice` configuration value will be ignored.
 
     `gasreq`, together with `gasprice`, will contribute to determining the penalty provision set aside by Mangrove from the market maker's `balanceOf` balance.
 
   An offer cannot be inserted in a closed market, nor when a reentrancy lock for `outbound_tkn`,`inbound_tkn` is on.
 
-  No more than $2^{32}-1$ offers can ever be created for one `outbound_tkn`,`inbound_tkn`, `tickSpacing` offerList.
+  No more than $2^{32}-1$ offers can ever be created for one (`outbound_tkn`,`inbound_tkn`, `tickSpacing`) offerList.
```

Comment:
```suggestion
  No more than $2^{32}-1$ offers can ever be created for one (`outbound`,`inbound`, `tickSpacing`) offerList.
```

Reply:
I am not sure when we reference using outbound_tkn or when we use outbound :)

Reply:
Ah yes it's because now there is `olKey.outbound` (not `olKey.outbound_tkn`). wdyt?